src/ui/tutorial/tutorial.js

```diff
@@ -589,6 +590,30 @@ cwc.ui.Tutorial.prototype.jumpToStep_ = function(stepID) {
   }
 };
 
+
+/**
+ * Scrolls the tutorial to the top of the given step
+ * @param {number} stepID
+ * @private
+ */
+cwc.ui.Tutorial.prototype.scrollToStep_ = function(stepID) {
+  if (stepID === undefined) {
+    stepID = this.getActiveStep_().id;
+  }
+  let rootNode = goog.dom.getElement(this.prefix + 'container');
```

Comment:
This line is duplicated, maybe defining something like this.rootNode = ... during the creation.